Candidates.txt

```diff
@@ -1,3 +1,2 @@
 Bogdan Djuric bogdan-djuric
-Aleksandra your name goes here - replace second line with your name
-Andrea Arsic Andrea-Arsic
\ No newline at end of file
+Andrea Arsic Andrea-Arsic
```

Comment:
It's not correct. In order to merge it correctly (with latest version of master) you should first pull latest changes from remote master branch to local master branch and than merge it to your branch.

Reply:
I pulled changes on my local branch and I hope it's ok now.

Reply:
That's fine, but you didn't merge this file as expected. You shouldn't remove some rows from that file, you should just add your at the end of this file (at next empty line)